src/parser.rs

```diff
@@ -77,15 +77,15 @@ pub(crate) fn parse_tests<'a>(test_str: &'a str) -> HashMap<String, Test<'a>> {
     tests
 }
 
-fn indent_level(lines: &Vec<&str>, line_off: usize) -> usize {
+fn indent_level(lines: &[&str], line_off: usize) -> usize {
```

Comment:
I've seen in the past Rust people stating that passing a slice over a `Vec` here more idiomatic. I can't seem to find the find the a link to the reasoning anymore though. Do you know why this is better?

Reply:
It's more flexible: you can do things like `&vec[n..m]` and so on. Is it useful here? Not really. But it's not unuseful, and it's slightly fewer characters ;)

Reply:
I guess a slice is more generic? It could be from a Vec or and array?

Reply:
Ok I'm convinced.